### ISSUE TITLE
Maintenance request: Outdated/deprecated npm packages

### ISSUE THREAD

**Issue by cypheron** [ripple/ripple-client#2655]
This project badly needs an upgrade. Lots of npm packages and bower itself are outdated as of today.
`│npm WARN deprecated jade@0.35.0: Jade has been renamed to pug, please install the latest ver
│ion of pug instead of jade
│npm WARN deprecated jade@1.11.0: Jade has been renamed to pug, please install the latest ver
│ion of pug instead of jade
│npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the gui
│elines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
│npm WARN deprecated transformers@2.1.0: Deprecated, use jstransformer
│npm WARN deprecated jade@0.28.2: Jade has been renamed to pug, please install the latest ver
│ion of pug instead of jade
│npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a R
│gExp DoS issue
│npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
│npm WARN deprecated graceful-fs@2.0.3: please upgrade to graceful-fs 4 for compatibility wit
│ current and future versions of Node.js
│WARN engine karma-cli@0.0.4: wanted: {"node":"~0.8 || ~0.10"} (current: {"node":"4.2.6","npm
│npm WARN deprecated gulp-webpack@1.5.0: Renamed to https://www.npmjs.com/package/webpack-str
│am
│WARN engine karma@0.12.37: wanted: {"node":">=0.8 <=0.12 || >=1 <=2"} (current: {"node":"4.2
│npm ERR! fetch failed https://registry.npmjs.org/camelcase/-/camelcase-3.0.0.tgz
│npm WARN retry will retry, error on last attempt: Error: connect EHOSTUNREACH 151.101.112.16
│:443
│npm ERR! fetch failed https://registry.npmjs.org/yargs-parser/-/yargs-parser-4.2.1.tgz
│npm WARN retry will retry, error on last attempt: Error: connect EHOSTUNREACH 151.101.112.162
│:443
│npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility wit
│h current and future versions of Node.js
│npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a R
│egExp DoS issue
│npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a R
│egExp DoS issue
│npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with
│ current and future versions of Node.js
│npm WARN deprecated gulp-util@3.0.1: gulp-util is deprecated - replace it, following the guid
│elines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
│npm WARN deprecated minimatch@1.0.0: Please update to minimatch 3.0.2 or higher to avoid a Re
│gExp DoS issue
│npm WARN deprecated gulp-util@2.2.20: gulp-util is deprecated - replace it, following the gui
│delines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
│npm WARN deprecated connect@2.30.2: connect 2.x series is deprecated
│npm WARN deprecated coffee-script@1.8.0: CoffeeScript on NPM has moved to "coffeescript" (no
│hyphen)N deprecated connect@2.30.2: connect 2.x series is deprecated
│npm WARN deprecated minimatch@0.4.0: Please update to minimatch 3.0.2 or higher to avoid a Re
│gExp DoS issue
│npm WARN deprecated bower@1.8.2: ...psst! Your project can stop working at any moment because
│ its dependencies can change. Prevent this by migrating to Yarn: https://bower.io/blog/2017/h
│ow-to-migrate-away-from-bower/`

**Comment by joshhazelhurst123** [comment]
Yes, I second that. Would be nice to have an update on what's happening.

**Comment by sublimator** [comment]
I believe it’s unmaintained

**Comment by zhenghejun** [comment]
I just can't not build the project.

Get these output after run cmd "gulp dist"
============================================================
events.js:167
      throw er; // Unhandled 'error' event
      ^
Error: CSS parse error js/head-f09d1a34.js: Unexpected input
============================================================

Have you ever meet this kind of problem?

**Comment by zhenghejun** [comment]
I just can't not build the project.

Get these output after run cmd "gulp dist"
```

events.js:167
throw er; // Unhandled 'error' event
^
Error: CSS parse error js/head-f09d1a34.js: Unexpected input

```
Have you ever meet this kind of problem?
